src/OrchardCore.Themes/TheAdmin/wwwroot/css/TheAdmin.css

```diff
@@ -7971,6 +7971,18 @@ input[type=password]::-ms-clear {
   padding: 0.75rem 0.5rem;
 }
 
+.EasyMDEContainer blockquote:not(.blockquote) {
+  background: var(--bs-secondary-bg-subtle);
+  border-left: 10px solid var(--bs-secondary-border-subtle) !important;
```

Comment:
Reduce the left border

> This is an example

Reply:
@hishamco unless the feedback address an issue with the code, I am not willing to make a change. If you think it is important, please download the branch, fix the spacing "as your soul desire" and then commit the change directly yo the branch.

Reply:
From the above screenshot, I saw you fixed the issue but the border is too much, I don't want to delay your PR because I have many things to do before I get on this, if you are not in a hurry I will do it

Reply:
`10px` is what is used as a default value by many styles. Anyway, I slightly reduced it to `0.4rem`.

![image](https://github.com/OrchardCMS/OrchardCore/assets/24724371/5b43a947-fad6-4376-9652-e1a753da47de)

Reply:
> 10px is what is used as a default value by many styles

Which styles you are referring to?

Reply:
I meant theme. here is an example of using 10px. https://www.markdownguide.org/basic-syntax/#blockquotes-1

but it does not matter as we can have our own value and it is a personal preference at the end of the day.